src/main/kotlin/com/wire/apps/polls/setup/EnvConfigVariables.kt

```diff
@@ -21,14 +21,4 @@ object EnvConfigVariables {
      * `jdbc:postgresql://localhost:5432/app-database`
      */
     const val DB_URL = "DB_URL"
-
-    /**
-     * Token which is used for the auth of proxy.
-     */
-    const val SERVICE_TOKEN = "SERVICE_TOKEN"
```

Comment:
These 2 values are still present in the README, remove from there also and check that the README does not contain any other deprecated information